test/Spec.hs

```diff
@@ -192,9 +192,11 @@ compilerTest progress java fname cli_args crate_types = do
         compileFlatBufferBindings prog specname (dir </> rustProjectDir specname)
     -- compile generated Java code
     classpath <- (maybe "" (":" ++ )) <$> lookupEnv "CLASSPATH"
+    path <- (maybe "" (":" ++ )) <$> lookupEnv "PATH"
```

Comment:
Can be just `path <- (maybe "" id) <$> lookupEnv "PATH"`, as we are not appending anything to it.